testing/tests/DevExpress.ui.widgets/treeViewParts/expandedItems.js

```diff
@@ -187,6 +187,22 @@ module('Expanded items', {
         });
     });
 
+    test('item with custom expander icons should expand on click', function(assert) {
+        const data = $.extend(true, [], DATA[5]);
+        const $treeView = initTree({
+            items: data,
+            expandButtonIcon: 'add',
+            collapseButtonIcon: 'add',
+        });
+        const treeViewInstance = $treeView.dxTreeView('instance');
+        const $firstItem = $treeView.find('.' + internals.ITEM_CLASS).eq(0);
+        const $icon = $firstItem.parent().find('> .' + internals.CUSTOM_EXPAND_ICON_CLASS);
```

Comment:
1. Cannot we simplify this icon getter?
As far as i see, u have a special class for custom icons, cannot we use it here?
2. We usually use template literals instead of a concatenation - it's just easier to read

Let's update everywhere in your tests